src/uint.rs

```diff
@@ -16,9 +16,6 @@ use crate::{
     Width,
 };
 
-#[cfg(feature = "rand_support")]
-use rand;
```

Comment:
Why remove `rand` support here? Same for `int.rs`.

Reply:
It was redundant because the place it was being used in already had a `#[cfg(feature = "rand_support")]` on it